docs/registry/advanced/iterative-development.md

```diff
@@ -36,7 +36,12 @@ If you are developing a module for the same target architecture as your developm
    If you are using a programming language that does not require compilation, such as Python, you can skip this step.
 
 1. Navigate to the Viam app, select your machine, and [add your module as a local module](/registry/configure/#local-modules) to your machine.
-   Provide the **Executable path** in the configuration, pointing to the compiled or built binary, or the executable script, depending on your language.
+   For the **Executable path** field, enter the absolute path on your machine's filesystem to either:
+
+   - the module's [executable file](/registry/create/#compile-or-package-your-module), such as `run.sh` or a compiled binary.
+   - a packaged tarball of your module, ending in `.tar.gz` or `.tgz`.
```

Comment:
Would it be worthwhile to link to documentation here about how to create a packaged tarball? I can't find a great link but something like [this](https://www.cs.swarthmore.edu/~newhall/unixhelp/howto_tar.html) could be helpful

Reply:
```suggestion
   - a [packaged tarball](https://www.cs.swarthmore.edu/~newhall/unixhelp/howto_tar.html) of your module, ending in `.tar.gz` or `.tgz`.
```